### DISCUSSION TITLE
How to download without converting?

### DISCUSSION THREAD

**Discussion by Michel-0** [aclap-dev/video-downloadhelper#1179] [Help and Q&A]
### Your question

Hi,

I'm highly confused about the watermark thing - or misunderstanding it.

[https://www.downloadhelper.net/about-licensing](https://www.downloadhelper.net/about-licensing)
> You may have noticed a sign at the corner of the videos after they have been processed **by the converter**. This is related to the fact the **conversion feature** has not been registered in the add-on.

How to download a video and **NOT** *converting* it?
I just wanted to download a video **as it is** without any conversion, how to do that?
Tried different offered `Download` buttons and options... all resulting videos got the watermark on it.

If it isn't possible at all to download without a watermark... I highly recommend to change your description at above mentioned webpage.

Kind regards,
Michel-0

---
version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
```JSON
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\<SNIP>\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\<SNIP>\\dwhelper"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 7,
    "successfull_dl": 2
}
```

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\<SNIP>\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\<SNIP>\\dwhelper"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 7,
    "successfull_dl": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

If you do not have a license & you are getting the watermark, that's the way it works.  If you don't want the watermark, you have to buy a license.  There are types of downloads that don't get a watermark.  I don't know what those are.  I have had a license for so long I almost forget I have one.  But your question is a bit sneaky.  The purpose of the watermark is to encourage you to buy a license.  If you don't like the watermark, your only solution  is to buy a license.  Otherwise, you just have to put up with it.  There is no way to circumvent this.  Buy a license or put up with it.  It's a simple as that.

**Reply by Michel-0** [reply]
> If you don't want the watermark, you have to buy a license.

That's not what they are saying[^1].

> There are types of downloads that don't get a watermark. I don't know what those are.

That's basically my question here. As I understand their statement[^1], the types are *simply* **unconverted**.

> The purpose of the watermark is to encourage you to buy a license. If you don't like the watermark, your only solution is to buy a license.

I am ok with that. But I am not ok with it if they are advertising it _otherwise_. **What they are doing**.
If it actually is like that - they should say it like that...

Looking forward to more technical information about download format possiblilities.

[^1]: [https://www.downloadhelper.net/about-licensing](https://www.downloadhelper.net/about-licensing)

**Reply by GrampaWildWilly** [reply]
Ah.  The VDH web site.  It's been a sore spot for several years.  I think Paul & Jérôme have cleaned it up to a certain extent recently.  But there are still corners of it that are woefully out of date.  I think the page you are referring to is one such bit of obsolete blather.  For example, the terminology "conversion license" is no longer current.  Any reference to a "converter" is obsolete.  This terminology was applicable in the days before VDH acquired the CoApp.  That would be when the latest version of VDH was 6 or older.  Starting with VDH 7, there is just a VDH license.  I've been using VDH for about 5 or 6 years now.  That actually does not make me an old timer.  There's people in here who have been using it much longer.  But you can take my word for it that conversion is just not a step that makes much sense in the world of the current VDH 9.  Things are done so much differently today.

The purpose of the watermark is, as I said, to encourage you to buy a license.  It kind of sort of says that on the page you are referring to.  But you have to understand something.  The VDH guys are French, based in France.  Their native language is not English.  They are pretty good with English but here & there you'll read something expressed in a way that a native English speaker would not say it.  If I were writing what it says on that page, I'd tweak a few things.  Minor tweaks, but tweaks just the same.

It's unfortunate that you've been misled by this obsolete web page on the VDH web site.  The simple state of affairs today is that the license tells VDH to skip the writing of the watermark into your downloaded content.  Conversion is just no longer a relevant topic here.  It is my understanding that there are certain types of download that don't get a watermark even if you don't have a license.  But I've had a license forever so I don't know what those are.  Maybe somebody else can come in here & list them.  There was such a list on our prior forum on Google Groups.  But that list was made a couple of years ago.  VDH works so much differently now that I'm not sure that list is still applicable.  Again, it would take somebody else posting here to give you a detailed list.

But it would be much simpler if you just bought a license.  The license is for life.  You can use it on every computer you own now, & every computer you ever own in the future.  So the longer you have a license, the cheaper it gets.  There is no renewal, annual maintenance fee, or any other sort of recurring charge.  You buy the license, that's it.  A one-time outlay of cash & you're done forever.

Understand that the license is tied to a particular browser: Firefox, Chrome, Edge.  Those are the only supported browsers.  So if you use 2 browsers, like say Firefox & Chrome, you need 2 licenses.  But you would still need only one instance of a particular license, which you can use on all your computers.
